packages/testkit-backend/src/skipped-tests/rx.js

```diff
@@ -4,6 +4,15 @@ const skippedTests = [
   skip(
     'Throws after run insted of the first next because of the backend implementation',
     ifEquals('stub.disconnects.test_disconnects.TestDisconnects.test_disconnect_on_tx_begin')
+  ),
+  skip(
+    'Backend could not support this test',
```

Comment:
what happened with these?

Reply:
But is there a reasonable way for users to handle this? Because there sure should be a way how users can handle unknown tz ids coming from the server.

Reply:
It should have, I will double check it again.

Reply:
It was actually the driver log causing issues. When the debug is turned on, the driver tries to access the broken object and it moves the point where the error occurs (it happens earlier, before the user get access to the record). 

I made the broken object a bit more lenient and I added a especial serialisation for this kind of object in the `json.stringify` wrapper.